.companion.rc

```diff
@@ -13,7 +16,7 @@ for default_param_file in *; do
 done
 
 if [ ! -f /home/pi/.updating ]; then
-	sudo -H -u pi screen -dm -S mavproxy $COMPANION_DIR/tools/telem.py
+	sudo -H -u pi screen -L -dm -S mavproxy $COMPANION_DIR/tools/telem.py
```

Comment:
You can use `-Logfile` to create a file with a better name.

Reply:
You can also put it in a better place, like /tmp. I don't know if we need this file after each boot, if yes you can use a place better than /tmp.

Reply:
screen has an option to log everything?

Reply:
yes! It does.

Reply:
No. now it's not. so i will remove it.